pkg/resolution/resolver/framework/testing/fakecontroller.go

```diff
@@ -169,3 +171,26 @@ func ensureConfigurationConfigMapsExist(d *test.Data) {
 		})
 	}
 }
+
+// CreateResolutionRequestStatusWithData returns a ResolutionRequestStatus with the resolved content.
+func CreateResolutionRequestStatusWithData(content []byte) *v1beta1.ResolutionRequestStatus {
```

Comment:
I'm not sure if having those files in the `fakecontroller.go` file is good or not. 😂 Or they could be in a new separate file? Those functions seem not controller related...?

Reply:
SGTM! Created a new file named `resolutionrequest.go`, and they are now living in this new file :D PTAL.

Reply:
👍

Reply:
I hope I could have the lgtm permission 😂